glossary.scss

```diff
@@ -0,0 +1,158 @@
+@import "common";
+
+.glossary {
+  padding: 24px 0;
+  font-family: "Aclonica", sans-serif;
+
+  .title {
+    padding-top: 10px;
+    margin: 0 20px 24px;
+    border-top: 1.5px solid $color-block-divider;
+
+    img {
+      max-width: 114px;
+    }
+  }
+
+  .slider {
+    background: linear-gradient(45deg, #E1E1E1, $color-primary);
+    padding: 24px 20px;
+
+    .list {
+      position: relative;
+      background: $color-primary;
+      box-shadow: 0 12px 24px $color-mark;
+      border-radius: 5px;
+      @include fontProp(16px, 32px, $color-secondary-text);
+
+      ul {
+        padding: 39px 0 41px 62px;
```

Comment:
use paired indents here 40px, do not use perfect pixel for this design, please try to do your project consistently and easily supportable

Reply:
done